lib/hacluster.pm

```diff
@@ -1219,7 +1219,7 @@ sub cluster_status_matches_regex {
     my $previous_line = '';
 
     for my $line (split("\n", $show_cluster_status)) {
-        if ($line =~ /(Stopped:| Stopped|Failed:| Failed|Pending:| Pending|Blocked:| Blocked)/) {
+        if (lc($line) =~ /\s?(stopped|failed|pending|blocked|starting|promoting):?/) {
```

Comment:
Suggestion:
```
if ($line =~ /\s?(stopped|failed|pending|blocked|starting|promoting):?/i) {
```
`i` means insensitive :)

Reply:
Oh.. this is what we were searching for yesterday! :D

Reply:
https://regex101.com/r/FIrl0n/1 and the ticket has some comment with explanation about it